src/Microsoft.ML.Dnn/ImageClassificationTransform.cs

```diff
@@ -186,6 +186,8 @@ private void CheckTrainingParameters(ImageClassificationEstimator.Options option
 
             if (_session.graph.OperationByName(_labelTensor.name.Split(':')[0]) == null)
                 throw Host.ExceptParam(nameof(options.TensorFlowLabel), $"'{options.TensorFlowLabel}' does not exist in the model");
+            if (options.EarlyStoppingCriteria != null && options.ValidationSet == null && options.TestOnTrainSet == false)
+                throw Host.ExceptParam(nameof(options.EarlyStoppingCriteria), $"No Validation dataset provided and testing on Train disabled, Early Stopping not supported.");
```

Comment:
>"No Validation dataset provided and testing on Train disabled, Early Stopping not supported. [](start = 79, length = 92)

Lets improve the exception message "Early stopping enabled but unable to find a validation set and/or train set testing disabled. Please disable early stopping or either provide a validation set or enable train set training."